pkg/agent/agent.go

```diff
@@ -87,6 +88,10 @@ func (k *Klocksmith) Run(stop <-chan struct{}) {
 // process performs the agent reconciliation to reboot the node or stops when
 // the stop channel is closed.
 func (k *Klocksmith) process(stop <-chan struct{}) error {
+	if err := k.waitForPodWithLabel(); err != nil {
+		return fmt.Errorf("failed to wait for pod with label: %v", err)
```

Comment:
one thing we should be doing in this code is using  the go 1.13 errors methods. I think. vs doing this propagation. Don't think we should refactor everything for it now, but we could start with the code we add. In this example using errors.Wrap() instead fo fmt.Errorf

Reply:
https://blog.golang.org/go1.13-errors

Reply:
Good idea, I forgot about the new `errors.Wrap()` stuff. Fixed.